### DISCUSSION TITLE
reflow mode for primereact `datatable`

### DISCUSSION THREAD

**Discussion by nimo23** [primefaces/community#3113] [PrimeReact] [answered]
Please add reflow mode for primereact `datatable` component to support responsive datatable (like the reflow mode supported in http://www.primefaces.org/showcase/ui/data/datatable/responsive.xhtml?jfwid=a12af)

**Comment by melloware** [comment, marked as answer]
Read this thread on why PrimeTek removed it in PrimeReact and see through the thread for someone who posted a workaround to make it work: https://github.com/primefaces/primereact/issues/4078
